commons/ava_networks/n_node_gecko_network.go

```diff
@@ -25,23 +25,25 @@ func (network NNodeGeckoNetwork) GetGeckoClient(i int) (gecko_client.GeckoClient
 type NNodeGeckoNetworkLoader struct{
 	numNodes int
 	numBootNodes int
+	isStaking bool
 }
 
-func NewNNodeGeckoNetworkLoader(numNodes int, numBootNodes int) (*NNodeGeckoNetworkLoader, error) {
+func NewNNodeGeckoNetworkLoader(numNodes int, numBootNodes int, isStaking bool) (*NNodeGeckoNetworkLoader, error) {
 	if numBootNodes > numNodes {
```

Comment:
Since we only have one cert right now, we shouldn't allow users to create a network if isStaking and numBootNodes > 1, else we'll get boot nodes with the same cert. Can you add a check for that, and a TODO explaining why this is and when we can get rid of this check?

Reply:
oh wow i had this exact check here but i forgot to push it. we wont get boot nodes with the same cert, what we'll get is bootnodes with different certs but we won't know their IDs so we cant connect to them

Reply:
done